.github/workflows/enable-pack-release.yaml

```diff
@@ -0,0 +1,123 @@
+        # checkout the latest on the branch instead of the sha from when triggered
+        ref: ${{ github.ref }}
+        fetch-depth: 1
+
+    - name: Checkout ci repo
+      uses: actions/checkout@v2
+      with:
+        repository: StackStorm-Exchange/ci
+        ref: ${{ env.ci_branch }}
+        path: ci
+        fetch-depth: 1
+
+    - name: Checkout exchange-tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: StackStorm-Exchange/exchange-tools
+        ref: ${{ env.exchange_tools_branch }}
+        path: exchange-tools
+        fetch-depth: 1
+
+    - name: Checkout Pack Repos and enable their Release Workflows
+      shell: bash
+      env:
+        GH_TOKEN: ${{ github.token }}
+        PACKS_ORG: ${{ env.packs_org }}
+        PACKS_PREFIX: ${{ env.pack_repo_prefix }}
+      run: |
+        mkdir -p ${PACKS_PATH}
+        cd ${PACKS_PATH}
+        source ${TOOLS_DIR}/functions.sh
+        EXCLUDE_PACKS=$(grep -v '^#' "${INDEX_DIR}/v1/exclude_packs.txt" | xargs echo -n)
+        for repo_name in $(_gh_list_repo_names ${PACKS_ORG} ${PACKS_PREFIX}); do
+          echo "::group::Clone ${PACKS_ORG}/${repo_name}"
+          gh repo clone "${PACKS_ORG}/${repo_name}"
+          if latestTag=$(git -C ${repo_name} describe --tags `git -C ${repo_name} rev-list --tags --max-count=1`); then
+            echo latestTag = $latestTag
+            git -C ${repo_name} checkout $latestTag -b latestTagBranch
+          fi
+          echo
+          echo "::endgroup::" # DELETED notices will not be folded to simplify scanning action output
+          if [[ -z "$latestTag" ]]; then
+            rm -rf ${repo_name}
+            echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing git tags"
+            echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+          elif [[ ! -f "./${repo_name}/pack.yaml" ]]; then
+            rm -rf ${repo_name}
+            echo "DELETED clone of ${PACKS_ORG}/${repo_name}: missing pack.yaml"
+            echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+          elif [[ " ${EXCLUDE_PACKS} " =~ " ${repo_name#${PACKS_PREFIX}-} " ]]; then
+            rm -rf ${repo_name}
+            echo "DELETED clone of ${PACKS_ORG}/${repo_name}: pack excluded via index.git/v1/exclude_packs.txt"
+            echo "${PACKS_ORG}/${repo_name}" >> ${GITHUB_WORKSPACE}/bad_pack_repos
+          else
+            #
+            #  This is not a bad or excluded Pack, ensure the Release workflow is active
+            #
+            echo "::group::Ensure Release workflow for ${PACKS_ORG}/${repo_name} is active"
+            if [[ -f "./${repo_name}/.github/workflows/release.yaml" ]]; then
+              echo "GitHub Release workflow manifest found for ${PACKS_ORG}/${repo_name}"
+                # NOTE: Unfortunately "gh workflow list" does not have --json
+                if ! gh workflow list ${PACKS_ORG}/${repo_name} --all | grep -E '^Release\b\s+active\b'; then
```

Comment:
If `gh workflow list` isn't giving good output, you could use `gh` to make the API calls more directly too.
Translating your old curl command would be something like: `gh api -X GET repos/{owner}/{repo}/actions/workflows/release.yaml --jq .state`

Reply:
> If `gh workflow list` isn't giving good output, you could use `gh` to make the API calls more directly too. Translating your old curl command would be something like: `gh api -X GET repos/{owner}/{repo}/actions/workflows/release.yaml --jq .state`

Ooh, that's nice, thanks.